src/celengine/meshmanager.cpp

```diff
@@ -96,6 +96,10 @@ cmod::Mesh
 ConvertTriangleMesh(const M3DTriangleMesh& mesh,
                     const M3DScene& scene)
 {
+    // Some community addons ships buggy meshes.
+    if (mesh.getMeshMaterialGroupCount() == 0)
```

Comment:
Would also make sense to abort processing in the case where `nFaces` or `nVertices` are zero